README.md

```diff
@@ -463,6 +463,12 @@ grunt
 
 The built version (data-layer-helper.js) will be in the `dist/` subdirectory.
 
+This library is being modernized, and the tests are being moved. You can run the new tests
```

Comment:
Seems like the sentence trails off a bit, maybe "You can run the new tests with the command:" ?

Reply:
Fixed, also I needed to add a step about installing it globally.